makefile

```diff
@@ -170,11 +170,15 @@ $(buildDir)/build-cross-compile:scripts/build-cross-compile.go makefile
 $(buildDir)/make-tarball:scripts/make-tarball.go $(buildDir)/render-gopath
 	$(vendorGopath) go build -o $@ $<
 dist:$(buildDir)/dist.tar.gz
+# don't cross-compile agents for a single-architecture environment, like in testing
+dist-single-arch:$(buildDir)/dist-single-arch.tar.gz
 dist-test:$(buildDir)/dist-test.tar.gz
 dist-race: $(buildDir)/dist-race.tar.gz
 dist-source:$(buildDir)/dist-source.tar.gz
 $(buildDir)/dist.tar.gz:$(buildDir)/make-tarball plugins clis $(binaries) $(clientBuildDir)/version
 	./$< --name $@ --prefix $(name) $(foreach item,$(binaries) $(distContents),--item $(item))
+$(buildDir)/dist-single-arch.tar.gz:$(buildDir)/make-tarball plugins cli $(binaries) $(clientBuildDir)/version
+	./$< --name $@ --prefix $(name) $(foreach item,$(binaries) $(distContents),--item $(item))
```

Comment:
I'm pretty sure that all client/cli builds will be made into this tarball, but because of the dependencies, only one will be rebuilt, which would be confusing